source/SUBMIT.rst

```diff
@@ -289,9 +287,18 @@ you can easily test your preprint build locally.
 
 **Step 2 - Data**
 
-If your executable content depends on a dataset, please make sure that your notebooks have access to it.
+First, make sure that your data is available online.
```

Comment:
--> First, make sure that your data is available online and can be downloaded publicly.